bigquery_etl/pytest_plugin/script_lint/pydocstyle.py

```diff
@@ -2,18 +2,18 @@
 
 import re
 
-from pytest_docstyle import Item
+from pytest_pydocstyle import Item
 
 from . import is_python_executable
 
 
 # adapted from
-# https://github.com/henry0312/pytest-docstyle/blob/v1.5.0/pytest_docstyle.py#L35-L41
+# https://github.com/henry0312/pytest-docstyle/blob/v2.0.0/pytest_pydocstyle.py#L34-L42
 def pytest_collect_file(parent, path):
     """Collect docstyle tests."""
     config = parent.config
     if (
-        config.getoption("docstyle")
+        config.getoption("pydocstyle")
         and is_python_executable(path)
         # https://github.com/PyCQA/pydocstyle/blob/2.1.1/src/pydocstyle/config.py#L163
         and re.match(config.getini("docstyle_match"), path.basename)
```

Comment:
This is causing throwing errors now. Perhaps @relud understands how to get past this.